backend/business-partner-agent/src/main/java/org/hyperledger/bpa/config/AcaPyConfig.java

```diff
@@ -34,42 +35,42 @@
 @Slf4j
 public class AcaPyConfig implements ApplicationEventListener<StartupTasks.AcaPyReady> {
 
+    @JsonIgnore
     @Inject
-    AriesClient ac;
+    transient AriesClient ac;
 
-    Boolean autoRespondCredentialOffer;
-    Boolean autoRespondCredentialProposal;
-    Boolean autoRespondCredentialRequest;
-    Boolean autoRespondPresentationProposal;
-    Boolean autoRespondPresentationRequest;
-    Boolean autoVerifyPresentation;
-    Boolean autoStoreCredential;
-    Boolean autoAcceptInvites;
-    Boolean autoAcceptRequests;
-    Boolean autoRespondMessages;
+    private Boolean autoRespondCredentialOffer;
```

Comment:
how about this?

making all switches `boolean autoAcceptInvites=false`
and read the config like this:
`Boolean.TRUE.equals(c.getUnwrapped("debug.auto_accept_invites", Boolean.class))`

this would make consuming the config easier, as long as configuration absence is interpreted as false.

Reply:
Good idea. I'm only hesitant to set default values because that can change semantics (in theory), but as aca-py does not return a value in the config if the flag is set to false the meaning stays the same anyway. Also it was working before, but like this it is cleaner and also keeps working if this behavior should change.